wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession+RecurringAction.swift

```diff
@@ -90,7 +90,8 @@ extension ZMUserSession {
 
             Task {
                 do {
-                    try await self.e2eiRepository.fetchFederationCertificates()
+                    let e2eiRepository = await self.viewContext.perform { self.e2eiRepository }
```

Comment:
fix coredata multithread access

Reply:
Is there a way we could have a precondition to check if these core data values are accessed on the correct context?

Reply:
replaced